source/lac/slepc_solver.cc

```diff
@@ -195,22 +195,28 @@ namespace SLEPcWrappers
       for (unsigned int i = 0; i < *n_converged; i++)
         {
           double residual_norm_i = 0.0;
-          // EPSComputeResidualNorm is L2-norm and is not consistent with the stopping criteria
-          // used during the solution process.
-          // Yet, this is the norm which gives error bounds (Saad, 1992, ch3):
-          //   | \lambda - \widehat\lambda | <= ||r||_2
-          ierr = EPSComputeResidualNorm (eps, i, &residual_norm_i);
-
-          // EPSComputeRelativeError may not be consistent with the stopping criteria
-          // used during the solution process. Given EPS_CONV_ABS set above,
-          // this can be either the l2 norm or the mass-matrix induced norm
-          // when EPS_GHEP is set.
-          // ierr = EPSComputeRelativeError (solver_data->eps, i, &residual_norm_i);
-
-          // EPSGetErrorEstimate is consistent with the residual norm
-          // used during the solution process. However, it is not guaranteed to
-          // be derived from the residual even when EPSSetTrueResidual is set.
-          // ierr = EPSGetErrorEstimate (solver_data->eps, i, &residual_norm_i);
+          // EPSComputeError (or, in older versions of SLEPc,
+          // EPSComputeRelativeError) uses an L2-norm and is not consistent
+          // with the stopping criterion used during the solution process (see
+          // the SLEPC manual, section 2.5). Since EPS_CONV_ABS is set above,
```

Comment:
"Since" sentence previously applied to EPSComputeRelativeError, i don't think it holds for `EPSComputeError`